draft-ietf-webtrans-http2.md

```diff
@@ -251,368 +288,393 @@ unidirectional, indicated by the second least significant bit of the
 
    Application Protocol Error Code:
-   : A variable-length integer containing the
-   application protocol error code that indicates why the stream is being
-   closed.
+   : A variable-length integer containing the application protocol error code
+     that indicates why the stream is being closed.
 
-Unlike the equivalent QUIC frame, this frame does not include a Final Size
-field. In-order delivery of WT_STREAM frames ensures that the amount of
+Unlike the equivalent QUIC frame, this capsule does not include a Final Size
+field. In-order delivery of WT_STREAM capsules ensures that the amount of
 session-level flow control consumed by a stream is always known by both
 endpoints.
 
-## WT_STOP_SENDING Frames {#WT_STOP_SENDING}
+## WT_STOP_SENDING Capsule {#WT_STOP_SENDING}
 
 *[WT_STOP_SENDING]: #
 
-A WebTransport frame called WT_STOP_SENDING is introduced to communicate that
-incoming data is being discarded on receipt per application request.
-WT_STOP_SENDING requests that a peer cease transmission on a stream.
+An HTTP capsule {{HTTP-DATAGRAM}} called WT_STOP_SENDING (type=0x190B4D3A) is
+introduced to communicate that incoming data is being discarded on receipt per
+application request. WT_STOP_SENDING requests that a peer cease transmission on
+a WebTransport stream.
 
 ~~~
-WT_STOP_SENDING Frame {
-  Type (i) = 0x05,
+WT_STOP_SENDING Capsule {
+  Type (i) = 0x190B4D3A,
   Length (i),
   Stream ID (i),
   Application Protocol Error Code (i),
 }
 ~~~
-{: #fig-wt_stop_sending title="WT_STOP_SENDING Frame Format"}
+{: #fig-wt_stop_sending title="WT_STOP_SENDING Capsule Format"}
 
-The WT_STOP_SENDING frame defines the following fields:
+The WT_STOP_SENDING capsule defines the following fields:
 
    Stream ID:
-   : A variable-length integer carrying the WebTransport stream ID of
-   the stream being ignored.
+   : A variable-length integer carrying the WebTransport stream ID of the stream
+     being ignored.
 
    Application Protocol Error Code:
-   : A variable-length integer containing the
-   application-specified reason the sender is ignoring the stream.
+   : A variable-length integer containing the application-specified reason the
+     sender is ignoring the stream.
 
 
-## WT_STREAM Frames {#WT_STREAM}
+## WT_STREAM Capsule {#WT_STREAM}
 
 *[WT_STREAM]: #
 
-WT_STREAM frames implicitly create a stream and carry stream data.
+WT_STREAM capsules implicitly create a WebTransport stream and carry stream
+data.
 
-The Type field in the WT_STREAM frame is either 0x0a or 0x0b.  This uses the
-same frame types as a QUIC STREAM frame with the OFF bit clear and the LEN bit
-set.  The FIN bit (0x01) in the frame type indicates that the frame marks the
-end of the stream in one direction.  Stream data consists of any number of 0x0a
-frames followed by a terminal 0x0b frame.
+The Type field in the WT_STREAM capsule is either 0x190B4D3B or 0x190B4D3C. This
+uses the same capsule types as a QUIC STREAM frame with the OFF bit clear and
```

Comment:
I don’t believe that is true with the new numbers

Reply:
Whoops, good catch

Reply:
Fixed in [167cad5](https://github.com/ietf-wg-webtrans/draft-webtransport-http2/pull/58/commits/167cad5b8fbcc1df3093bf2ec83fe598bdf6886c)